brands/shop/electronics.json

```
@@ -1,4 +1,13 @@
 {
+   "shop/electronics|ABC_Warehouse": {
```

Looks mostly good! - can you remove the underscore from the name here?
```suggestion
   "shop/electronics|ABC Warehouse": {
```

---

@bhousel - I have removed underscore from the name. Thanks.